tasties_app/views.py

```diff
@@ -61,3 +61,14 @@ def register(request):
 
     context = {'form': form}
     return render(request, 'tasties_app/register.html', context)
+
+
+@login_required(login_url='login')
+def single_recipe(request, recipe=None):
+    if not recipe:
+        recipe = Recipe.objects.first()
```

Comment:
This is temporary, I know, but risky... Maybe worth your effort to put a comment over this line saying it should be amended in the future.

Reply:
I'll comment but it's gonna change very soon. Working on it.

Reply:
Why not redirect to somewhere else in case there is no recipe?

Reply:
Changes the method to redirect to the `recipes` page if no recipe is given.